Makefile

```diff
@@ -108,8 +108,9 @@ else
 	$(call MINIKUBE_GO_BUILD_CMD,$@,$*)
 endif
 
-out/e2e-%-amd64:
-	GOOS=$* GOARCH=amd64 go test -c k8s.io/minikube/test/integration --tags=integration -o $@
+.PHONY: e2e-%-amd64
+e2e-%-amd64:
```

Comment:
I couldn't figure out how to get the dependencies for the integration test package.  I tried to follow the same method as localkube, minikube, but I wasn't getting the full dependency set.  I think it had something to do with the fact that they were `_test.go` files.